modules/Lending.py

```diff
@@ -73,10 +74,13 @@ def init(cfg, api1, log1, data, maxtolend, dry_run1, analysis, notify_conf1):
 
     sleep_time = sleep_time_active  # Start with active mode
 
-    if notify_conf['enable_notifications'] and notify_conf['notify_summary_minutes']:
+    if notify_conf['enable_notifications']:
```

Comment:
In retrospect we don't actually need to check `if notify_conf['enable_notifications']:` at all. To make things more clear we could check `if scheduler.empty():` before creating and starting the thread.